otlp/.gitattributes

```diff
@@ -1 +1,2 @@
+proto/**/*.proto linguist-generated=true
```

Comment:
@michaelpj I added this line to hide the diffs for the protobuf files too.
You can have a look here https://github.com/marinelli/hs-opentelemetry/pull/1, just to be sure it works as expected.